README.md

```diff
@@ -241,15 +241,61 @@ CKB> account list
   ledger_id: 0x69c46b6dd072a2693378ef4f5f35dcd82f826dc1fdcc891255db5870f54b06e6
 ```
 
-The `ledger_id` shown is the public key hash for the path m/44'/309', which is the root Nervos path. Remember the `ledger\_id` given above for the latest commands. It will be
-used for ```<ledger-id>``` later on.
+The `ledger_id` shown is the public key hash for the path m/44'/309', which is the root Nervos path. the `ledger\_id` will be
+used for ```<ledger-id>``` argument in the `account import` command as described below.
 
-### Get Public Key ###
+If you have already imported the Ledger account, then `account list` command will instead give the account details.
+This will be shown even if the device is not connected.
 
-Get the public key:
+``` sh
+CKB> account list
+- "#": 0
+  account_source: ledger hardware wallet
+  address:
+    mainnet: ckb1qyqry754h4tevmngdll9jrpnrnfhqqhpcccskdl3en
+    testnet: ckt1qyqry754h4tevmngdll9jrpnrnfhqqhpcccstgpw40
+  lock_arg: 0x327a95bd57966e686ffe590c331cd37002e1c631
+  lock_hash: 0xc27b9ad3414cf5b1720713663d5f754e8968793f2da90b6428feb565bf94de4e
+```
+
+## Import Ledger Wallet account ###
+
+Use the `account import --ledger <ledger_id>` command to import the account to the `ckb-cli`.
+You will receive a confirmation prompt on the device which should say `Import Account` and `Account 1`.
+Confirm this to import the account. This operation will provide the extended root public key (m/44'/309'/0') to the `ckb-cli`.
+
+``` sh
+CKB> account import --ledger 0x69c46b6dd072a2693378ef4f5f35dcd82f826dc1fdcc891255db5870f54b06e6
+- "#": 0
+  account_source: ledger hardware wallet
+  address:
+    mainnet: ckb1qyqry754h4tevmngdll9jrpnrnfhqqhpcccskdl3en
+    testnet: ckt1qyqry754h4tevmngdll9jrpnrnfhqqhpcccstgpw40
+  lock_arg: 0x327a95bd57966e686ffe590c331cd37002e1c631
+  lock_hash: 0xc27b9ad3414cf5b1720713663d5f754e8968793f2da90b6428feb565bf94de4e
+```
+
+### Get BIP44 Wallet Public Keys ###
+
+Use the `account bip44-addresses` command to obtain the BIP44 addresses for the mainnet.
+
+Note that this command is provided as a convenience by the `ckb-cli` to get a list of addresses with the derivation path quickly.
+
+**It is highly recommended to verify the account provided by this command on the Ledger device using the `account extended-address` command as described next.**
 
 ``` sh
-CKB> account extended-address --path "m/44'/309'/0'/1/0" --account-id <ledger-id>
+CKB> account bip44-addresses --lock-arg 0x327a95bd57966e686ffe590c331cd37002e1c631
+```
+
+### Obtain / Verify Public Key ###
+
+The `account extended-address` command should be used to
+
+- Verify the public key obtained via `account bip44-addresses` command on the Ledger device
+- Obtain the public key for any arbitrary BIP44 derivation path
+
+``` sh
+CKB> account extended-address --path "m/44'/309'/0'/1/0" --lock-arg 0x327a95bd57966e686ffe590c331cd37002e1c631
```

Comment:
Let's not have our example be a change address, let's make it a receiving address. That is more likely the type of address a user should be confirming : `m/44'/309'/0'/0/0`

Reply:
We should update the `Derivation Path` in the code blob below to match this, too

Reply:
Our spec also no longer shows `Mainnet Address` on the prompt - it should just be `Address` - let's update that in the code blob below as well

Reply:
That means it isn't necessary to show the full section on 'Testnet Address', but we should still reference the on-device configuration